fury/shaders/tests/test_base.py

```diff
@@ -88,6 +140,15 @@
     """
 
 
+def generate_cone():
+    cone_src = ConeSource()
+    cone_mapper = PolyDataMapper()
+    cone_mapper.SetInputConnection(cone_src.GetOutputPort())
+    cone_actor = Actor()
+    cone_actor.SetMapper(cone_mapper)
+    return cone_actor
```

Comment:
Why restart from scratch and not use an actor?   

To reduce the maintenance, you should try to reduce as much as possible code duplication

Reply:
The test using this function was moved from `test_shader.py` to `test_base.py`. I have updated it by using `actor.cone` instead.